scico/__init__.py

```diff
@@ -48,6 +48,8 @@
     "custom_vjp",
 ]
 
+from . import random, linop
```

Comment:
This is unavoidable?

Reply:
Not needed. This should either be removed or expanded to cover other modules. It slightly bothers me that one can't, e.g., 
```python
import scico
x, _ = scico.random(...)
```

Reply:
Worth discussing.

Reply:
Removed.